MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1182,14 +1181,18 @@ private static int GetMaxAudioBitrateForTotalBitrate(long totalBitrate)
                     }
 
                     // Check audio codec
-                    var selectedAudioStream = candidateAudioStreams.FirstOrDefault(audioStream => directPlayProfile.SupportsAudioCodec(audioStream.Codec));
-                    if (selectedAudioStream == null)
+                    MediaStream selectedAudioStream = null;
+                    if (candidateAudioStreams.Any())
```

Comment:
Looks like `candidateAudioStreams` is iterated over multiple times, maybe it schould be a collection instead of an `IEnumerable?`

Reply:
Seems like a good idea, I'll look into it tomorrow.

Reply:
Done. Let me know if I missed anything.